lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -141,18 +141,24 @@ class _StartOpenSearch(EventBase):
     This event will be deferred until OpenSearch starts.
     """
 
-    def __init__(self, handle, *, ignore_lock=False, after_upgrade=False):
+    def __init__(self, handle, *, ignore_lock=False, after_upgrade=False, first_data_node=False):
```

Comment:
to differentiate between the `first_data_node` field
```suggestion
    def __init__(self, handle, *, ignore_lock=False, after_upgrade=False, is_first_data_node=False):
```